### PR TITLE
updated sample and pipeline process name

### PR DESCRIPTION
**Pull request by andyk-ms**
Updated AutomaticDeviceManagementSample to show exception handling for RemoveConfigurationAsync commonly used in the service SDK

Updated SDL package name in pipeline process

### CODE REVIEW

File: vsts/vsts.yaml [hunk 1]
```diff
@@ -629,7 +629,7 @@ jobs:
           tsaVersion: 'TsaV2'
           codebase: 'NewOrUpdate'
           tsaEnvironment: 'PROD'
-          codeBaseName: 'Azure-Iot-SDK-CSharp-Master'
```
Comment (andyk-ms) [inline vsts/vsts.yaml:632]: No longer refer to our code base with this name.

File: iothub/service/samples/how to guides/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs [hunk 2]
```diff
@@ -90,23 +101,45 @@ private static void CreateMetricsAndTargetCondition(Configuration configuration)
...
             foreach (Configuration configuration in configurations)
             {
+                num += 1;
                 string configurationString = JsonConvert.SerializeObject(configuration, Formatting.Indented);
                 Console.WriteLine(configurationString);
                 Thread.Sleep(1000);
             }
 
-            Console.WriteLine("Configurations received");
+            Console.WriteLine("Configurations received: " + num);
```
Comment (drwill-ms) [inline iothub/service/samples/how to guides/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs:142]:
```suggestion
            Console.WriteLine($"Configurations received: {num}.");
```

File: vsts/vsts.yaml [hunk 3]
```diff
@@ -629,7 +629,7 @@ jobs:
...
+          codeBaseName: 'Azure-Iot-SDK-CSharp'
           notificationAlias: 'AzIoTDnDMgdSdk@microsoft.com'
```
Comment (drwill-ms) [inline vsts/vsts.yaml:633]:
```suggestion
          notificationAlias: 'AzIoTEdMgdSdk@microsoft.com'
```

**Review by drwill-ms — APPROVED** [review]

**Review by patilsnr — APPROVED** [review]

**Review by abhipsaMisra — APPROVED** [review]

**Pull request merged**